integration_go.mod

```diff
@@ -0,0 +1,91 @@
+module gopkg.in/DataDog/dd-trace-go.v1 // This file is manually maintained and updated
```

Comment:
we should probably change this module name to be `github.com/DataDog/dd-trace-go/v2` to match the rest of the mod files (see the `go.mod` file in this directory).